### PR TITLE
builtin: get_format function implemented.

### PR DESCRIPTION
**Pull request by Tratar**

### CODE REVIEW

File: parser/parser.y [hunk 1]
```diff
@@ -3085,6 +3087,13 @@ FunctionCallNonKeyword:
 			Args: []ast.ExprNode{$3.(ast.ExprNode), $5.(ast.ExprNode)},
 		}
 	}
+|	"GET_FORMAT" '(' GetFormatSelector ','  Expression ')'
+	{
+		$$ = &ast.FuncCallExpr{
+            FnName: model.NewCIStr($1),
```
Comment (tiancaiamao) [inline parser/parser.y]: ditto

File: expression/builtin_time.go [hunk 2]
```diff
@@ -1010,6 +1012,68 @@ func (b *builtinFromUnixTimeSig) eval(row []types.Datum) (d types.Datum, err err
...
+}
+
+func (b *builtinGetFormatSig) eval(row []types.Datum) (d types.Datum, err error) {
+	args, err := b.evalArgs(row)
+	t := args[0].GetString()
+	l := args[1].GetString()
+	switch t {
+	case "DATE":
+		switch l {
+		case "USA":
```
Comment (shenli) [inline expression/builtin_time.go]: Could you define some consts for these string literals?

**Comment by shenli** [comment]
@Tratar Please fix the conflicts.

**Comment by Tratar** [comment]
Done

**Comment by tiancaiamao** [comment]
LGTM

**Comment by coocood** [comment]
@Tratar 
Thank you for your contribution!

**Pull request merged**